ui/src/app/components/buttons.tsx

```diff
@@ -21,6 +21,18 @@ const styles = {
     backgroundColor: disabled ? '#f1f2f2' : (hover ? '#262262' : '#ffffff'),
     borderRadius: '0.2rem',
     color: disabled ? '#c3c3c3' : (hover ? '#ffffff' : '#262262'),
+  }),
+  darklingPrimary: ({ hover, disabled }) => ({
```

Comment:
currently i don't believe we use these buttons anywhere else in the site, but making this general in case we decide to have more modals that use a dark background